services/port_manager/src/main/java/com/futurewei/alcor/portmanager/util/NetworkConfigurationUtil.java

```diff
@@ -137,7 +137,8 @@ public static NetworkConfiguration buildNetworkConfiguration(List<Object> entiti
                 }
 
                 if (!subnetUniqueIds.contains(subnetId)) {
-                    Long tunnelId = subnetEntity.getTenantId() !=null ? Long.valueOf(subnetEntity.getTenantId()): null;
+                    // FIXME ：subnetEntity.getVpcId().hashCode() need to be changed to segmentId
+                    Long tunnelId = subnetEntity.getTenantId() !=null ? Long.valueOf(getHashCode(subnetEntity.getVpcId())): null;
```

Comment:
It seems that we didn't format this file using Ctrl+Shift+L :-)

Reply:
This line of code is quite long and a bit buggy. Let us move it to an util method with appropriate check.

Reply:
NP